pgcrypto_expressions/fields.py

```diff
@@ -45,24 +45,30 @@ def __init__(self, *args, **kwargs):
         self.base_db_type = super(
```

Comment:
I know it's not strictly related to the feature in this PR, but it is closely related to the code reorg that's happening here: would you be willing to move the calculation of `base_db_type` also into a method that takes the connection? And then that method can be called from `DecryptedCol.as_sql`, which is the only place the base type is needed. Currently this calculation of `base_db_type` is technically broken because it always uses the default connection; it would break if the default connection is not Postgres. (Thus the `@@@` comment above.)

(I'm fine not having a test to specifically exercise the default-db-is-not-PG case, I'd just like to get the code rearranged to handle it correctly, which is possible now that you've done the rest of the reorg for multi-db here.)